oval_graph/converter.py

```diff
@@ -52,18 +52,24 @@ def get_tag(self):
             return str(self.tree.tag)
         return ""
 
+    def get_result(self):
+        return self.tree.evaluate_tree() if self.tree.evaluate_tree() else self.tree.value
```

Comment:
I recommend to not evaluate it twice. Save the result to a variable. By the way, the reason why `evaluate_tree` doesn't always return the desired result isn't obvious. Perhaps, it requires a comment in the source code.

Reply:
@Honny1 What about this?